### PR TITLE
[Conf] Add SenderScore RPBL return codes

### PR DESCRIPTION
**Pull request by moisseev**

### CODE REVIEW

**Comment by moisseev** [comment]
@dragoangel Could you please review/test this configuration, with a particular focus on the weights?

**Comment by dragoangel** [comment]
Will do, thank you

**Comment by dragoangel** [comment]
Also I thinking about `noauth` thing, we already have scores for all sorts of `AUTH_NA` and so on, so I think - lets put `noauth` zero score, and where it added as part of something - do not add due to it `0.5`?

**Comment by dragoangel** [comment]
Also after deeper investigation I can say that 
https://multirbl.valli.org/detail/score.senderscore.com.html has much better spam detection then https://multirbl.valli.org/detail/bl.score.senderscore.com.html

As Validity at https://knowledge.validity.com/hc/en-us/articles/20961730681243 states about 10k queries per day for free (also asking to register IPs of resolvers) - we better utilize only 1 by default, I propose enable `score.senderscore.com` by default and put `bl.score.senderscore.com` as disabled.

**Review by dragoangel — CHANGES_REQUESTED** [review]

File: conf/modules.d/rbl.conf [hunk 1]
```diff
@@ -82,6 +82,26 @@ rbl {
       symbol = "RBL_SENDERSCORE";
```
[the remark this answers is not in the document]
Comment (moisseev) [reply conf/modules.d/rbl.conf:82]: I understand the reasoning behind `RBL_SENDERSCORE_UNKNOWN`. However, in my opinion, it’s better to keep `RBL_SENDERSCORE` without the `_UNKNOWN` suffix to stay consistent with the naming conventions used for other RBLs.

What do you think?

**Pull request merged**